recipe/patches/0000-fix-classifier-clash-in-pyproject.patch

```diff
@@ -0,0 +1,9 @@
+--- pyproject.toml
```

Comment:
what's the reason for the patch?

Reply:
I'm curious as to why things in the classifier section cause issues for us. If it is needed, some more info in the patch header would be useful

Reply:
Been addressed with a comment, results in a mangled metadata. I've done a lot of testing and I'm happy to address with another solution